firebaseAdmin.ts

```
@@ -0,0 +1,11 @@
+import * as admin from "firebase-admin";
+
+if (!admin.apps.length) {
+  admin.initializeApp({
+    credential: admin.credential.cert(
+      JSON.parse(process.env.FIREBASE_ADMIN_SDK_JSON || "{}")
```

Similar to something Nick said when I initially added SendGrid, but maybe we can add a check here instead of falling back to {} (e.g. throwing an error if the env variable isn't defined). I'm not sure how initializeApp works under the hood but adding the check might cause the program to fail faster without attempting any Firebase operations that are done in initializeApp, and the error message might be more specific/user-friendly than whatever Firebase throws

---

100% agreed, will change!